app/src/main/java/nl/eduvpn/app/service/OrganizationService.kt

```diff
@@ -148,7 +150,7 @@ class OrganizationService(private val serializerService: SerializerService,
         }
     }
 
-    private suspend fun createGetJsonSingle(url: String): String {
+    private suspend fun createGetJsonSingle(url: String): Pair<ByteArray, Charset> {
```

Comment:
This should return a string. If you need a call which returns some a byte array, please add a new method, or rename this one to make it clear (if it is not called on other places)